WinUIGallery/Styles/TitleBar.xaml

```diff
@@ -0,0 +1,285 @@
+
+                        <ContentPresenter
+                            x:Name="PART_LeftContentPresenter"
+                            Grid.Column="3"
+                            x:DeferLoadStrategy="Lazy"
+                            Content="{TemplateBinding LeftContent}"
+                            Visibility="Collapsed" />
+
+                        <Viewbox
+                            x:Name="PART_Icon"
+                            Grid.Column="5"
+                            MaxWidth="16"
+                            MaxHeight="16"
+                            Margin="0,0,16,0"
+                            VerticalAlignment="Center"
+                            x:DeferLoadStrategy="Lazy"
+                            Child="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IconElement}"
+                            Visibility="Collapsed" />
+
+                        <TextBlock
+                            x:Name="PART_TitleText"
+                            Grid.Column="6"
+                            MinWidth="48"
+                            Margin="0,0,8,0"
+                            HorizontalAlignment="Left"
+                            VerticalAlignment="Center"
+                            x:DeferLoadStrategy="Lazy"
+                            Foreground="{TemplateBinding Foreground}"
+                            Style="{StaticResource CaptionTextBlockStyle}"
+                            Text="{TemplateBinding Title}"
+                            TextTrimming="CharacterEllipsis"
+                            TextWrapping="NoWrap"
+                            Visibility="Collapsed" />
+
+                        <TextBlock
+                            x:Name="PART_SubtitleText"
+                            Grid.Column="7"
+                            MinWidth="48"
+                            Margin="0,0,16,0"
+                            HorizontalAlignment="Left"
+                            VerticalAlignment="Center"
+                            x:DeferLoadStrategy="Lazy"
+                            Foreground="{ThemeResource TitleBarSubtitleForegroundBrush}"
+                            Style="{StaticResource CaptionTextBlockStyle}"
+                            Text="{TemplateBinding Subtitle}"
+                            TextTrimming="CharacterEllipsis"
+                            TextWrapping="NoWrap"
+                            Visibility="Collapsed" />
+
+                        <Grid
+                            x:Name="PART_CenterContentPresenterGrid"
+                            Grid.Column="8"
+                            x:DeferLoadStrategy="Lazy"
+                            Visibility="Collapsed">
+                            <ContentPresenter
+                                x:Name="PART_CenterContentPresenter"
+                                HorizontalAlignment="Center"
+                                Content="{TemplateBinding CenterContent}" />
```

Comment:
Is it `Content` or `CenterContent`?

Reply:
This is fixed now in the latest commit; `Content` was chosen as the property for hosting content in the center.